molecule_ec2/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -118,7 +121,8 @@
           'user': "{{ ssh_user }}",
           'port': "{{ ssh_port }}",
           'identity_file': "{{ keypair_path }}",
-          'instance_ids': "{{ item.instance_ids }}", }
+          'instance_ids': "{{ item.instance_ids }}",
+          'region': "{{ item.instances[0].region if item.instances[0].region is defined else omit }}", }
```

Comment:
this can be failed, because sometimes instances list is empty and API returns the instance_ids in tagged_instances list.

Reply:
maybe, we can have something like 
```'instance_ids': "{{ item.instance_ids
          if item.instance_ids | bool
          else [item.tagged_instances[0].id] }}",```

Reply:
Thanks for the suggestion @mohitkumarsharmaflux7 I can certain see that being an issue, I didn't touch that line with the exception of adding a "," to extend it to add region. Should this still be in the same PR?

Reply:
I would say, let's keep it simple for now, we can cover that edge case later. I will open a new PR or that. Thanks